src/components/back-link/index.md.njk

```diff
@@ -11,6 +11,8 @@ layout: layout-pane.njk
 
 Use the back link component to help users go back to the previous page in a multi-page transaction.
 
+Although browsers have a back button, some sites break when you use it - so many users avoid it. Also, not all users are aware of the back button.
```

Comment:
```suggestion
Although browsers have a back button, some sites break when you use it - so many users avoid it, instead of losing their progress in a service. Also, not all users are aware of the back button.
```